_layouts/post.html

```diff
@@ -5,20 +5,16 @@
   {% include navbar/index.html %}
```

Comment:
It doesn't look like you've tested these changes as the icons and not showing on smaller devices. Please revert all changes here and the ones in CSS related to the positioning and animation. If you want you can tweak my work after I have completed #GH-60 because if this is merged at the moment it will be more complicated to fix twitter as I wouldn't be able to test it on mobile phones.